gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/ResultQueueTest.java

```diff
@@ -281,7 +281,7 @@ public void shouldDrainAsItemsArrive() throws Exception {
                 latch.countDown();
             });
 
-            assertThat(latch.await(3000, TimeUnit.MILLISECONDS), is(true));
+            assertThat(latch.await(10000, TimeUnit.MILLISECONDS), is(true));
```

Comment:
any specific reason for the increase...were you just getting some timeouts here?

Reply:
not need to revert if it was causing hassles. just curious.